src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/PluginIT.java

```diff
@@ -52,22 +55,399 @@ public void sqlEnableSettingsTest() throws IOException {
         }
 
         queryResult = new JSONObject(TestUtils.getResponseBody(response));
-        Assert.assertThat(queryResult.getInt("status"), equalTo(400));
+        assertThat(queryResult.getInt("status"), equalTo(400));
         JSONObject error = queryResult.getJSONObject("error");
-        Assert.assertThat(error.getString("reason"), equalTo("Invalid SQL query"));
-        Assert.assertThat(error.getString("details"), equalTo("Either opendistro.sql.enabled or rest.action.multi.allow_explicit_index setting is false"));
-        Assert.assertThat(error.getString("type"), equalTo("SQLFeatureDisabledException"));
-        resetClusterSettings(PERSISTENT, "opendistro.sql.enabled");
+        assertThat(error.getString("reason"), equalTo("Invalid SQL query"));
+        assertThat(error.getString("details"), equalTo("Either opendistro.sql.enabled or rest.action.multi.allow_explicit_index setting is false"));
+        assertThat(error.getString("type"), equalTo("SQLFeatureDisabledException"));
+        wipeAllClusterSettings();
+    }
+
+    @Test
```

Comment:
not a big deal, just curious- is there a way we can export some of these JSON strings to a separate file?

Reply:
I initially thought of putting it in some file under `tests/resources` directory, but reason of keeping here in the test suite is to avoid the breaking the workflow for the reviewer to go back and forth between tests and fixtures file or any developer looking at it without IDE.